packages/core/src/main/getDiForUnitTesting.ts

```diff
@@ -50,6 +49,8 @@ export function getDiForUnitTesting() {
     for (const block of chunk(100)(injectables)) {
       di.register(...block);
     }
+
+    di.register(kubectlVersionsInjectable);
```

Comment:
This registration is the responsibility of the `kubrectl-versions` feature. Introduce `feature.ts` in the package (check example in the `application` -feature) and use `registerFeature(kubectlFeature)`

Reply:
Ok, cool